### PR TITLE
📊 guardian: decadal averages

### PR DESCRIPTION
**Pull request by lucasrodes**
[Tracking issue](https://github.com/owid/owid-issues/issues/1445)

Add decadal averages.

### CODE REVIEW

File: etl/steps/data/garden/news/2024-05-08/guardian_mentions.py [hunk 1]
```diff
@@ -25,35 +46,86 @@ def run(dest_dir: str) -> None:
...
 
-    tb = tb.format(["country", "year"])
+    ## Add per-capita indicators
+    tb = geo.add_population_to_table(tb, ds_population)
+    for column in ["num_pages_tags", "num_pages_mentions"]:
+        tb[f"{column}_per_million"] = tb[column] / tb["population"] * 1_000_000
+    tb = tb.drop(columns="population")
 
+    # Estimate 10-year average
+    tb_10y_avg = tb[(tb["year"] >= 2014) & (tb["year"] <= 2023)].copy()
```
Comment (spoonerf) [inline etl/steps/data/garden/news/2024-05-08/guardian_mentions.py:60]: Maybe set these as global values at the top so it's easy to update in the future?

File: etl/steps/data/garden/news/2024-05-08/guardian_mentions.py [hunk 2]
```diff
@@ -25,35 +46,86 @@ def run(dest_dir: str) -> None:
...
 
     ## Get relative values
-    tb = add_relative_indicator(tb, "num_pages_tags")
-    tb = add_relative_indicator(tb, "num_pages_mentions")
+    tb = add_relative_indicators(tb, ["num_pages_tags", "num_pages_mentions"])
 
-    tb = tb.format(["country", "year"])
+    ## Add per-capita indicators
+    tb = geo.add_population_to_table(tb, ds_population)
+    for column in ["num_pages_tags", "num_pages_mentions"]:
```
Comment (spoonerf) [inline etl/steps/data/garden/news/2024-05-08/guardian_mentions.py:55]: You could consider excluding countries with less than a million people, as their values are easily inflated by relatively few mentions e.g. Greenland [here](http://staging-site-news/admin/charts/7905/edit)

File: etl/steps/data/garden/news/2024-05-08/guardian_mentions.py [hunk 3]
```diff
@@ -25,35 +46,86 @@ def run(dest_dir: str) -> None:
...
     ds_garden = create_dataset(
-        dest_dir, tables=[tb], check_variables_metadata=True, default_metadata=ds_meadow.metadata
+        dest_dir, tables=tables, check_variables_metadata=True, default_metadata=ds_meadow.metadata
     )
 
     # Save changes in the new garden dataset.
     ds_garden.save()
 
 
+def add_relative_indicators(tb, colnames):
```
Comment (spoonerf) [inline etl/steps/data/garden/news/2024-05-08/guardian_mentions.py:95]: Remember to set 'types' for the function arguments e.g. `tb:Table` - that's still best practice right?

File: snapshots/news/2024-05-07/guardian_mentions.py [hunk 4]
```diff
@@ -1,5 +1,7 @@
 """Getting the snapshot data requires interaction with The Guardian's Open Platform API (https://open-platform.theguardian.com/access/).
 
+NOTE: The data has been cosolidated into a Google Spreadhseet here https://docs.google.com/spreadsheets/d/18xjcsEjT7zTGTSNffju_zoD95RFxYl0vYKXc8jErKGk/edit#gid=0
```
Comment (spoonerf) [inline snapshots/news/2024-05-07/guardian_mentions.py:3]: Tiny typo in 'Spreadhseet'

**Review by spoonerf — APPROVED** [review]
Looks good! Could be worth dropping out countries with a small population e.g. Greenland, but up to you!

File: etl/steps/data/garden/news/2024-05-08/guardian_mentions.py (hunk 2 above)
Comment (lucasrodes) [reply etl/steps/data/garden/news/2024-05-08/guardian_mentions.py:55]: yeah... that makes sense ー  will leave for now, and see how useful this is for Max' project.

**Pull request merged**